docs/core/testing/selective-unit-tests.md

```diff
@@ -87,12 +88,55 @@ In the code example, the defined traits with keys `Category` and `Priority` can
 | Expression | Result |
 | ---------- | ------ |
 | `dotnet test --filter XUnit` | Runs tests whose `FullyQualifiedName` contains `XUnit`.  Available in `vstest 15.1+`. |
-| `dotnet test --filter Category=bvt` | Runs tests which have `[Trait("Category", "bvt")]`. |
+| `dotnet test --filter Category=CategoryA` | Runs tests which have `[Trait("Category", "CategoryA")]`. |
+
+**Using conditional operators | and &amp;**
+
+| Expression | Result |
+| ---------- | ------ |
+| <code>dotnet test --filter "FullyQualifiedName~TestClass1&#124;Category=CategoryA"</code> | Runs tests which has `TestClass1` in `FullyQualifiedName` **or** `Category` is `CategoryA`. |
+| `dotnet test --filter "FullyQualifiedName~TestClass1&Category=CategoryA"` | Runs tests which has `TestClass1` in `FullyQualifiedName` **and** `Category` is `CategoryA`. |
+| <code>dotnet test --filter "(FullyQualifiedName~TestClass1&Category=CategoryA)&#124;Priority=1"</code> | Runs tests which have either `FullyQualifiedName` containing `TestClass1` **and** `Category` is `CategoryA` **or** `Priority` is 1. |
+
+## NUnit
+
+```csharp
+using NUnit.Framework;
+
+namespace NUnitNamespace
+{
+    [TestClass]
```

Comment:
TestClass is the MSTest attribute here. `[TestFixture]` would be the NUnit equivalent, but it's not actually required in NUnit anymore - so leaving it off might be better! 🙂 

Thanks for adding NUnit here - everything else looks good to me! 😄